### ISSUE TITLE
Screen reader is not announcing as "selected" for selected date in calendar.

### ISSUE THREAD

**Issue by msftedad** [ng-bootstrap/ng-bootstrap#4609]
### Bug description:

**Repro steps:**
1. Launch the URL : [Angular powered Bootstrap (ng-bootstrap.github.io)](https://ng-bootstrap.github.io/#/components/datepicker/examples)
2. Turn on Narrator/NVDA/JAWS.
3. TAB till "Basic Date Picker".
4. TAB till the current date in calendar.
5. USE Arrow key to navigate to different dates.
6. Press enter on date to select it, selected date will get highlighted.
7. Observe that Screen reader is not announcing as "selected" for selected date in calendar.

**Actual Result:**
Screen reader is not announcing as "selected" for selected date in calendar.

**Expected Result:**
After selecting any date from calendar, screen reader should announce the date as selected.

### Versions of Angular, ng-bootstrap and Bootstrap:
ng-bootstrap:v15.1.2

**Attachment:**

[Calendar date_JAWS (1).webm](https://github.com/ng-bootstrap/ng-bootstrap/assets/93735775/bd35ec3f-d7f0-442a-997e-ce00268ef080)

[Calendar date_NVDA (1).webm](https://github.com/ng-bootstrap/ng-bootstrap/assets/93735775/8d5a2577-c634-471f-a6c6-7837b5577288)

**Comment by maxokorokov** [comment]
Any ideas on how to fix this? 
Seems like `aria-selected="true"` with `role="gridcell"` is not well supported.
